resources/views/research.blade.php

```diff
@@ -2,6 +2,7 @@
 <html>
     <head>
         <title>Monty Hall - Research Page</title>
-    </head>
+	</head>
```

Comment:
This is a more general comment - not just for this file. There seems to be a weird mix of spaces and actual tabs in a lot these files. We should stick to tabs being 4 spaces instead of tab characters